github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -61,6 +61,27 @@ presubmits:
             memory: "8Gi"
           limits:
             memory: "8Gi"
+  - name: pull-project-infra-test-plugins
+    run_if_changed: plugins/.*
+    cluster: ibm-prow-jobs
+    optional: false
+    decorate: true
+    spec:
+      containers:
+      - image: quay.io/kubevirtci/bootstrap:v20210126-a12b6c0
+        command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-c"
+        - |
+          bazelisk test //plugins/...
```

Comment:
nit do we need a multiline element here?
```suggestion
        - "bazelisk test //plugins/..."
```

Reply:
Good catch, thanks! Removed, PTAL!